en/advanced/ai.md

```diff
@@ -0,0 +1,79 @@
+# AI functionality in JabRef
+
```

Comment:
Here, much text is missing. I think, you can just copy and paste from the blog post.

Reason: We try to collect (and update) documentation at docs.jabref.org - and not blog.jabref.org. Blog.jabref.org is "only" For advertising new features, not providing deep explanations. -- user-documentation should be self-contained.

Reply:
What text is missing there?

And what technical details I provided in blog post? I think most of the blog post consists of 1) showing new features, 2) tutorial (a crappy one) how to get an OpenAI API key.

Reply:
Sections from https://github.com/InAnYan/blog.jabref.org/blob/ai-1/_posts/2024-07-01-AI-chatting.md

- AI chat tab
- How does this work?
- How to get an OpenAI API key?
- AI preferences (at least the screenshot and maybe some intro text)